CONTRIBUTING.md

```diff
@@ -38,6 +38,60 @@ After you cloned your fork, create a new branch and implement the changes in the
 gh pr create
 ```
 
+## Recording fixtures for testing
+
+Most parts of `github-project` is tested using full integrations test, using fixtures for GraphQL requests and responses. You can see all the tests with their fixtures in [`test/recorded/`](test/recorded/).
+
+If you changed how `github-project` is working or added a feature that is not covered by the existing tests, you need to update the fixtures.
+
+We record the fixtures using a dedicated GitHub organization: [@github-project-fixtures](https://github.com/github-project-fixtures/). We are invite you to the organization so that you can record your own fixtures without setting up your own GitHub organization and app, just let us know.
```

Comment:
```suggestion
We record the fixtures using a dedicated GitHub organization: [@github-project-fixtures](https://github.com/github-project-fixtures/). We can invite you to the organization so that you can record your own fixtures without setting up your own GitHub organization and app, just let us know.
```